File: backend/osmosewebsite/models/team.py
from django.db import models


class TeamMember(models.Model):
    name = models.CharField(max_length=100)
    position = models.CharField(max_length=255)
    biography = models.TextField()
    picture = models.URLField()

    mailAddress = models.EmailField()

    researchGateURL = models.URLField(blank=True, null=True)
    personalWebsiteURL = models.URLField(blank=True, null=True)
    githubURL = models.URLField(blank=True, null=True)
    linkedinURL = models.URLField(blank=True, null=True)

    def __str__(self):
        return self.name

File: website/src/models/team.ts
import { Contact } from "./bibliography";

export interface TeamMember {
  id: string;
  contact: Contact;
  position: string;
  biography?: string;
  picture: string;

  mail_address?: string;

  research_gate_url?: string;
  personal_website_url?: string;
  github_url?: string;
  linkedin_url?: string;

  is_former_member?: boolean;
}
